src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String sayHello;
```

Comment:
Not informative variable `sayHello`

Reply:
Not informative variable `sayHello`. Let's rename it(for example to `greetingMessage`)